EIPS/eip-3009.md

```diff
@@ -0,0 +1,317 @@
+---
+eip: 3009
+title: `transferWithAuthorization` - Gas-Abstracted ERC20 transactions with EIP-712
+author: Peter Jihoon Kim (@petejkim), Kevin Britz (@kbrizzle), David Knott (@DavidLKnott)
+discussions-to: https://github.com/ethereum/EIPs/issues/3010
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-28
+requires: 20, 712
+---
+
+## Simple Summary
```

Comment:
The simple summary section should really only be 1-2 sentences. I recommend condensing this down into that size.

Reply:
Think of the Simple Summary section like the subject line of an email, or a sub-title on a forum.  In this case something like "A contract interface that allows transferring fungible assets via authorization."

Reply:
Thank you, will make the amendment